apps/web/pages/api/user/avatar.ts

```diff
@@ -18,7 +18,7 @@ async function getIdentityData(req: NextApiRequest) {
   const { username, teamname } = querySchema.parse(req.query);
 
   if (username) {
-    const user = await prisma.user.findUnique({
+    const user = await prisma.user.findFirst({
       where: { username },
```

Comment:
Same as above - if i create a user called "peer" inside an org i get peers gravitar logo/the first peer in the DB